test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/OutcodeGeneratorTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.lang.reflect.Constructor;
+import java.util.Random;
+
+import org.junit.Test;
+
+public class OutcodeGeneratorTest {
+
+    @Test
+    public void shouldCheckConstructorIsNotAccessible()
```

Comment:
Small point, but it is less noisy to just throw exception on each test method. Makes things easier to read.

I know one should never throw Exception, but this is a test :-)